docusign-connector-demo/processes/eSign.mod

```diff
@@ -0,0 +1,413 @@
+<elementInfo>
+    <language>
+        <name>read 
+documents</name>
+    </language>
+</elementInfo>
+' #txt
+st0 f21 520 298 112 44 -35 -15 #rect
+st0 f22 actionTable 'out=in;
+' #txt
+st0 f22 eventIdConfig "in.envelopeId" #txt
+st0 f22 timeoutConfig 'ACTION_AFTER_TIMEOUT=NOTHING
+EXCEPTION_PROCESS_START=
+TIMEOUT_SCRIPT=' #txt
+st0 f22 eventBeanClass "com.axonivy.connector.docusign.event.EnvelopeCompleted" #txt
+st0 f22 eventBeanConfig "demo=in.envelopeId" #txt
+st0 f22 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>wait for signers</name>
+    </language>
+</elementInfo>
+' #txt
+st0 f22 561 193 30 30 19 -8 #rect
+st0 f22 res:/webContent/icons/docusign.png?small #fDecoratorIcon
+st0 f23 977 305 30 30 0 15 #rect
+st0 f24 clientId 3d3d7114-4397-48c9-8378-931fc35885c3 #txt
+st0 f24 path /v2.1/accounts/{accountId}/envelopes #txt
+st0 f24 queryParams 'cdse_mode=;
+change_routing_order=;
+completed_documents_only=;
+merge_roles_on_draft=;
+' #txt
+st0 f24 templateParams 'accountId=;
+' #txt
+st0 f24 method POST #txt
+st0 f24 bodyInputType ENTITY #txt
+st0 f24 bodyObjectType com.docusign.esign.model.EnvelopeDefinition #txt
+st0 f24 bodyObjectMapping 'param.documents=[com.axonivy.connector.docusign.DocUtils.ofIvyFile(in.file)];
+param.emailSubject="Connecting Axon Ivy with DocuSign";
+param.recipients.signers=[ com.axonivy.connector.docusign.SignUtils.myself(ivy.session) ];
+param.status="sent";
+' #txt
+st0 f24 resultType com.docusign.esign.model.EnvelopeSummary #txt
+st0 f24 responseMapping 'out.envelopeId=result.envelopeId;
+' #txt
+st0 f24 responseCode ivy.case.customFields().stringField("envelopeId").set(out.envelopeId); #txt
+st0 f24 clientErrorCode ivy:error:rest:client #txt
+st0 f24 statusErrorCode ivy:error:rest:client #txt
+st0 f24 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>create envelope</name>
+    </language>
+</elementInfo>
+' #txt
+st0 f24 416 104 112 48 -51 -7 #rect
+st0 f25 clientId 3d3d7114-4397-48c9-8378-931fc35885c3 #txt
+st0 f25 path /v2.1/accounts/{accountId}/envelopes/{envelopeId}/documents/{documentId} #txt
+st0 f25 queryParams 'documents_by_userid=;
+recipient_id=;
+shared_user_id=;
+certificate=;
+encoding=;
+encrypt=;
+language=;
+show_changes=;
+watermark=;
+' #txt
+st0 f25 templateParams 'accountId=;
+envelopeId=in.envelopeId;
+documentId=in.documents.get(0).documentId;
+' #txt
+st0 f25 responseCode 'import java.io.InputStream;
+InputStream inputStream = response.getEntity().toType(InputStream.class);
+
+String document = org.apache.commons.io.FilenameUtils.removeExtension(in.documents.get(0).name)+".pdf";
+ivy.case.documents().add(document).write().withContentFrom(inputStream);
+
+
+/*ivy.log.info("file: "+result);
+String document = org.apache.commons.io.FilenameUtils.removeExtension(in.documents.get(0).name)+".pdf";
+ivy.case.documents().add(document).write().withContentFrom(result);
```

Comment:
@ivy-rew result is no more available. does it work with inputstream?

Reply:
works if you leave open api mode and set a type `java.io.File` in the result type